src/components/Modals/Login/Login.modal.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useContext } from 'react';
+import Alert from '@material-ui/lab/Alert';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import TextField from '@material-ui/core/TextField';
+import loginApi from '../../../utils/apis/login.api';
+import { AppContext } from '../../../utils/AppContext.provider';
+
+const LoginModal = (props) => {
+  const { open, handleClose } = props;
+  const [errorSession, setErrorSession] = useState(false);
+  const [username, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+  const { setSessionData } = useContext(AppContext);
+  const handleLocalClose = () => {
+    setErrorSession(false);
+    handleClose();
+  };
+  const handleSubmitSession = () => {
+    const loginResult = loginApi(username, password);
```

Comment:
```js

```

It's ok. You can use `async await` in order to resolve the promise as well inside of a `try catch` block, and the code will be more readable